### ISSUE TITLE
Forms on Android don't automatically advance with the "next" button

### ISSUE THREAD

**Issue by Brokkoly** [formio/formio.js#3953]
- On Android, the browser (Chrome 89.0.4389.105) tries to allow the user to auto-advance through forms, using the enter key to tab to the next field:
- ![image](https://user-images.githubusercontent.com/14946087/114199884-50479500-991a-11eb-8b02-3b643cfd1fa6.png)
- Using Form.io forms on Android (Same chrome version, Application: 7.0.42, Renderer: 4.12.4), I don't see that behavior: 
- ![image](https://user-images.githubusercontent.com/14946087/114200105-7cfbac80-991a-11eb-9aa0-df2a5fcf47c1.png)
Is this a Form.io issue, or is there some property that we would need to be applying to our fields in order for this to work? (I've already done a lot with tabindex trying to get this to work)

Thanks

**Comment by wag110894** [comment]
Hello @Brokkoly,
Is the first screenshot a form.io form? or just a general form?

**Comment by Brokkoly** [comment]
> Hello @Brokkoly,
> Is the first screenshot a form.io form? or just a general form?

The first screenshot is a general form, made in react (source code for those is [here](https://github.com/Brokkoly/BrokkolyBotFrontend/blob/b2fb69962711f69014c99180286f16bee17b2d7d/ClientApp/src/components/CommandRow.tsx#L133).
I referenced it because I had to do no special handling for that to happen (though it could be a feature of react).

Just now I googled for a simple form in jsfiddle to see if it was just a fluke and it seems like it is not a fluke. I got the same (correct) behavior [here](https://jsfiddle.net/travistidwell/e3uh9xrd/) on my phone.
